src/Apcu/bootstrap.php

```diff
@@ -1,48 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-use Symfony\Polyfill\Apcu as p;
-
-if (!extension_loaded('apc') && !extension_loaded('apcu')) {
-    return;
-}
-
-if (!function_exists('apcu_add')) {
-    if (extension_loaded('Zend Data Cache')) {
```

Comment:
this case is not covered by just requiring ext-apcu+PHP7.3

Reply:
Isn't this just to make use of a 'better' solution if the Zend Data Cache is available? It provides the same functions even when this isn't available.

Reply:
this allows using the apcu functions when only Zend Data Cache is installed.
maybe add a conflict rule with "ext-Zend Data Cache": "*"?

Reply:
I have added the conflict rule

Reply:
thanks. can you add it to the root composer.json also?